src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline

```diff
@@ -1319,6 +1299,49 @@ function samplePublishSeries() {
 }
 
 
+// [END sample_core]
+// [END sample]
+// tslint:disable-next-line:no-any
+
+samplePublishSeries();
+============== file: src/samples/v1/publish_series_request_test_write_to_file.js ==============
+// DO NOT EDIT! This is a generated sample ("Request",  "test_write_to_file")
+'use strict';
+
+// [START sample]
+// [START sample_core]
+
+const library = require('@google-cloud/library').v1;
+const util = require('util');
+const fs = require('fs');
+
+/** Testing write fields to files. */
+function samplePublishSeries() {
+  const client = new library.LibraryServiceClient();
+  const shelf = {};
+  const books = [];
+  const seriesUuid = {};
+  const request = {
+    shelf: shelf,
+    books: books,
+    seriesUuid: seriesUuid,
+  };
+  client.publishSeries(request)
+    .then(responses => {
+      const response = responses[0];
+      // testing writing bytes field.
+      var writeFile = util.promisify(fs.writeFile);
+      writeFile(`uuid_of_series_with_book_${response.bookNames[0]}.raw`, response.seriesUuid.seriesBytes, 'binary');
+      // testing writing string field.
+
+      writeFile('series_uuid_in_plain_text.txt', response.seriesUuid.seriesString);
```

Comment:
Expanding on my comment above about vertical whitespace: for example, here, I personally would find it more readable to have:

```
const response = ...;

// testing ...
var writeFile = ...
writeFile(...)

// testing ...
writeFile(...)
```

Reply:
The first blank line is hard to predict. Sometimes the output handling part is just a print statement, in which case it seems a blank line between the rpc call and the print statement is redundant.

For now, I change the templates so that every file output has an implicit newline.

Reply:
(a) However we deal with this, I'm fine doing it in a chaser PR if it's not trivial. But we should remember to do it (file an issue, etc)

(b) I think relying on the user to specify newlines is easier for us, and I can imagine in some situations it's what the user wants. However, I'm guessing (just a guess!) most of the time it would be a drag for the user and make their sample config more verbose. Let's think a bit more whether there's an easy way to automate the common case. (There may not be and we may indeed wind up relying on user-specified blank lines.)

Spitballing one idea: maybe in general, if a config statement gets translated to more than one line of code (like file output), those lines get an automatic blank line after. This is what you said you tried and were concerned with the extra newline before the closing brace. How about making the logic for the newline be like a join (so we avoid the trailing blank line)? In snippet pseudo-code:
```
@file_write_1 = 'doSomething()\ndoSomethingElse()\n' # etc....
@multi_line_blocks = [ file_write_1, file_write_2, ...] 
function mySample() {
  @'\n'.join(multi_line_blocks)
}
```

Reply:
It sounds reasonable to me. I am a little worried that the snip templates are not intelligent enough to accomplish this mission :(. Issue filed: https://github.com/googleapis/gapic-generator/issues/2706